dockerfiles/Dockerfile.heroku-20

```diff
@@ -1,5 +1,4 @@
 FROM heroku/heroku:20
-MAINTAINER hone
```

Comment:
vscode suggested this change, not sure why TBH but deleting it didn't break anything